File: discover_ruby/github_most_popular.rb
require 'HTTPClient'

extheaders = {
  'User-Agent' => 'Holberton_School',
  'Authorization' => 'token 63d7b0ceea3a07803ac4e808caaa08d51e619bac'
}
clnt = HTTPClient.new #request httpclient access

puts clnt.get_content('https://api.github.com/search/repositories?q=language:ruby&sort=stars&order=desc', query = nil, extheader = {})

File: discover_more_python/github_most_popular.py
import urllib2

request_headers = {
  'User-Agent': 'Holberton_School',
  'Authorization': 'token 27aa992fd7186938f589f8b6da49a7264dcc808a'
}

githuburl = 'https://api.github.com/search/repositories?q=language:python&sort=stars&order=desc'

req = urllib2.Request(githuburl, headers=request_headers)
response = urllib2.urlopen(req) # Call is made
print response.read() # Print response in standard output
